src/loan.rs

```diff
@@ -888,14 +900,50 @@ impl Lender0 {
     /// Interpret a loan request and performs lender logic.
     ///
     /// rate is expressed in usdt sats per btc, i.e. rate = 1 BTC / USDT
+    #[deprecated(note = "Use interpret_loan_request instead")]
```

Comment:
You might want to set the `version` attribute as well.

Reply:
Yep. When I wrote this I wasn't sure about what the next version would be, but we've introduced breaking changes with this PR so it should be `0.3`.